src/containers/SearchPage/components/SearchResults.tsx

```diff
@@ -6,103 +6,139 @@
  *
  */
 
-import { CSSProperties, useCallback, useId } from "react";
+import { CSSProperties, ReactNode, useCallback, useId } from "react";
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { ButtonV2 } from "@ndla/button";
-import { breakpoints, colors, mq, spacing } from "@ndla/core";
-import { Cross } from "@ndla/icons/action";
-import { Button, Spinner } from "@ndla/primitives";
-import { Heading, Text } from "@ndla/typography";
-import { ContentTypeBadge } from "@ndla/ui";
+import { Done } from "@ndla/icons/editor";
+import {
+  Heading,
+  Text,
+  Button,
+  Spinner,
+  CheckboxGroup,
+  CheckboxRoot,
+  CheckboxControl,
+  CheckboxIndicator,
+  CheckboxLabel,
+  CheckboxHiddenInput,
+} from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
 import SearchResultItem from "./SearchResultItem";
 import { SearchGroup, TypeFilter } from "../searchHelpers";
-import { ViewType } from "../searchTypes";
 
-const Wrapper = styled.section`
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-  gap: ${spacing.normal};
-  margin: ${spacing.medium} 0;
-  position: relative;
-`;
+const Wrapper = styled("section", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "medium",
+    marginBlock: "medium",
+  },
+});
 
-const HeaderWrapper = styled.hgroup`
-  display: flex;
-  align-items: center;
-  gap: ${spacing.small};
-`;
+const HeaderWrapper = styled("hgroup", { base: { display: "flex", alignItems: "center", gap: "medium" } });
 
-const ButtonsWrapper = styled.div`
-  display: flex;
-  gap: ${spacing.small};
-  flex-wrap: wrap;
-`;
+const PaginationWrapper = styled("div", {
+  base: {
+    display: "flex",
+    flexDirection: "column",
+    gap: "small",
+    alignItems: "center",
+  },
+});
+const ProgressBar = styled("div", {
+  base: {
+    width: "surface.xxsmall",
+    height: "1",
+    background: "stroke.subtle",
+  },
+});
 
-const PaginationWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.small};
-  align-items: center;
-`;
+const Progress = styled("span", {
+  base: { display: "block", background: "stroke.default", height: "2px", width: "min(var(--width),100%)" },
+});
 
-const ProgressBar = styled.div`
-  width: 200px;
-  height: 2px;
-  background: ${colors.brand.tertiary};
-  margin: 0 0 ${spacing.small};
-`;
+export const SearchResultsList = styled("ul", {
```

Comment:
Har vi ikke to forskjellige visninger for denne?

Reply:
hva mener du?